localization/replace/english/wc_title_replacements_l_english.yml

```diff
@@ -223,3 +223,14 @@
  
  d_zulmamwe:0 "Skullsplitter"
  d_zulmamwe_adj:0 "Skullsplitter"
+ 
+ d_freehold:0 "Drisburg"
+ d_freehold_adj:0 "Drisburgian" 
+ c_freehold:0 "Drisburg"
+ c_freehold_adj:0 "Drisburgian"
```

Comment:
This doesn't make sense being [Drisburg](https://warcraft.wiki.gg/wiki/Drisburg) because it's supposed to be on the northern coast of Kul Tiras, while Freehold is in the southmost coast.
It's also supposed to have gnome and dwarf immigrants from Khaz Modan, ruled by [Duke Falrevere](https://warcraft.wiki.gg/wiki/Duke_Falrevere) which also doesn't fit either on that regard.

Once the map gets bigger Kul Tiras should be big enough to have Drisburg as its own county.

Reply:
this branch is deprecated anyway, I'm just going to pick whatever was implemented and make GW a separate branch later